docs/cicd.md

```diff
@@ -0,0 +1,214 @@
+| ------------------------------ | ---------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------ | --------------------------------------- | ------------------------------------------------------------------------------------------------------ |
+| GOOGLE_SERVICE_ACCOUNT_JSON    | JSON for authentication in the Google Play Console as Release Manager. This should expire after two years. | Password Manager                                                         | {...}                                   | [Service Account Docu](https://cloud.google.com/iam/docs/creating-managing-service-account-keys?hl=de) |
+| CREDENTIALS_GIT_REPOSITORY_URL | Git remote URL to the credentials repository which contains the Java Keystore                              | Ask the team about this secret repository                                | git@github.com:User/credentials.git     | -                                                                                                      |
+| CREDENTIALS_DIRECTORY_PATH     | Path where the credentials Git repository cloned to automatically by FL                                    | The developer can choose this freely                                     | /home/circleci/credentials              | -                                                                                                      |
+| CREDENTIALS_KEYSTORE_PATH      | Path to the OpenSSL AES256-CBC encrypted Java Keystore file                                                | -                                                                        | /home/circleci/credentials/<secret>.enc | Look for the `openssl enc` command in the Android Fastlane file for more information                   |
+| KEYSTORE_PATH                  | Path to the decrypted Java Keystore file                                                                   | -                                                                        | /home/circleci/keystore.jks             | -                                                                                                      |
+| CREDENTIALS_KEYSTORE_PASSWORD  | Password for decrypting the keystore using OpenSSL                                                         |                                                                          | password                                | -                                                                                                      |
+| KEYSTORE_KEY_ALIAS             | Alias of the key within the Java Keystore                                                                  | You should look in the JKS file using `keytool -list -v -keystore <jks>` | my-key                                  | -                                                                                                      |
```

Comment:
this is specific per project and the env variables are called this:
KEYSTORE_KEY_ALIAS_BAYERN
KEYSTORE_KEY_ALIAS_NUERNBERG
KEYSTORE_KEY_ALIAS_KOBLENZ
KEYSTORE_KEY_PASSWORD_BAYERN
KEYSTORE_KEY_PASSWORD_NUERNBERG
'KEYSTORE_KEY_PASSWORD_KOBLENZ

Reply:
thx for the hint @ztefanie 
Is there any documentation about the gpg encryption. I think i didn't do a manual release after you created a general jks file and encrypted it. Maybe you can add sth to the app-credentials repo or give me some input :)